Eatery/Onboarding/Models/OnboardingModel.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  OnboardingModel.swift
+//  Eatery
+//
+//  Created by Reade Plunkett on 11/11/19.
+//  Copyright © 2019 CUAppDev. All rights reserved.
+//
+
+import UIKit
+
+class OnboardingModel {
```

Comment:
`struct`? This also feels a bit overkill to create a whole class to pass in three pieces of information to the `OnboardingViewController`